NextcloudTalk/LoginViewController.m

```diff
@@ -219,6 +244,70 @@ - (void)presentAuthenticationViewWithServerURL:(NSString *)serverURL withUser:(N
     [self presentViewController:navController animated:YES completion:nil];
 }
 
+#pragma mark - Files app accounts
+
+- (void)checkForFilesAppAccounts
+{
+    if (![NCUtils isNextcloudAppInstalled]) {
+        self.importButton.hidden = YES;
+        return;
+    }
+
+    NSString *path = [[[[NSFileManager defaultManager] containerURLForSecurityApplicationGroupIdentifier:appsGroupIdentifier] URLByAppendingPathComponent:kNextcloudFilesDatabaseFolder] path];
+    NSURL *accountsFileURL = [[NSURL fileURLWithPath:path] URLByAppendingPathComponent:kTalkAccountsFileName];
+    NSArray *filesAccounts = [[NKCommon shared] readDataAccountFileAt:accountsFileURL];
+    NSArray *talkAccounts = [[NCDatabaseManager sharedInstance] allAccounts];
+
+    _importedFilesAccount = [NSMutableArray new];
+    for (NKDataAccountFile *filesAccount in filesAccounts) {
+        BOOL accountIncluded = NO;
+        for (TalkAccount *talkAccount in talkAccounts) {
+            if ([talkAccount.server isEqualToString:filesAccount.url] && [talkAccount.user isEqualToString:filesAccount.user]) {
+                accountIncluded = YES;
+            }
+        }
+        if (!accountIncluded) {
+            [_importedFilesAccount addObject:filesAccount];
+        }
+    }
+
+    self.importButton.hidden = !_importedFilesAccount.count;
+}
+
+- (void)presentImportedAccountsSelector
+{
+    NSMutableArray *importedAccounts = [NSMutableArray new];
+    for (NKDataAccountFile *filesAccount in _importedFilesAccount) {
+        DetailedOption *option = [[DetailedOption alloc] init];
+        option.title = (!filesAccount.alias || [filesAccount.alias isEqualToString:@""]) ? filesAccount.user : filesAccount.alias;
```

Comment:
Not sure about this one:
```
(lldb) po filesAccount.user
marcel.mueller@nextcloud.com

(lldb) po filesAccount.alias
Marcel Müller
```

Because then the user is prepopulated with `Marcel Müller`, which doesn't make sense. Also this does break the Auth-Controller, it's spinning forever..
`DetailOption` has an `identifier` property, maybe always set that to `fileAccount.user` and use it for the login process?